### PR TITLE
[DOPS-2942]: Update substrate build

### PR DESCRIPTION
**Pull request by f33r0**
# Task

[DOPS-2942]: Update substrate build

## Changes

1. Updated env docker image (fixed build and coverage there)
2. Updated build workflow. Separated test step. Tests do not runs twice.

jenkins-library PR:
https://github.com/soramitsu/jenkins-library/pull/567

## Author

Signed-off-by: Nikita Zaporozhets <zaporozhets@soramitsu.co.jp>

[DOPS-2942]: https://soramitsu.atlassian.net/browse/DOPS-2942

### CODE REVIEW

**Review by wpi86 — APPROVED** [review]

**Review by C4tWithShell — APPROVED** [review]

File: Jenkinsfile [hunk 1]
```diff
@@ -1,14 +1,10 @@
...
-      envImageName: 'docker.soramitsu.co.jp/sora2/parachain-env:latest',
+      envImageName: 'docker.soramitsu.co.jp/sora2/env:env',
       appImageName: 'docker.soramitsu.co.jp/sora2/parachain',
       buildTestCmds: ['housekeeping/tests.sh'],
-      cargoClippyTag: ':latest',
-      cargoClippyCmds: ['housekeeping/clippy.sh'],
-      codeCoverage: false,
-      sonarProjectKey: 'sora:sora2-common',
-      sonarProjectName: 'sora2-common',
-      dojoProductType: 'sora'
```
Comment (C4tWithShell) [inline Jenkinsfile]: Will we get rid of dojo and sonar?

Comment (Alexey-N-Chernyshov) [reply Jenkinsfile]: Either I don't understand why sonar and defectDojo are removed. The same for code coverage.

File: housekeeping/tests.sh [hunk 2]
```diff
@@ -1,4 +1,12 @@
...
-cargo test  --release --features runtime-benchmarks
\ No newline at end of file
+test() {
+    cargo test  --release --features runtime-benchmarks
+}
+
+if [ "$(type -t $1)" = "function" ]; then
+    "$1"
+else
+    echo "Func '$1' is not exists in this workflow. Skipped."
```
Comment (Alexey-N-Chernyshov) [inline housekeeping/tests.sh]:
```suggestion
    echo "Func '$1' does not exist in this workflow. Skipped."
```

File: Jenkinsfile (hunk 1 above)
Comment (f33r0) [reply Jenkinsfile]: for tests. Reverted

**Pull request merged**